README.md

```diff
@@ -38,7 +38,7 @@ This repo contains all the code for data preprocessing, training and running inf
     * [7.1. Running QC on predictions from SCI-T2w dataset](#71-running-qc-on-predictions-from-sci-t2w-dataset)
     * [7.2. Running QC on predictions from MS-MP2RAGE dataset](#72-running-qc-on-predictions-from-ms-mp2rage-dataset)
     * [7.3. Running QC on predictions from Radiculopathy-EPI dataset](#73-running-qc-on-predictions-from-radiculopathy-epi-dataset)
-* [8. Active learning procedure](#8-active-learning-procedure-todo)
+* [8. Using contrast-agnostic model with nnUNet](#8-use-contrast-agnostic-model-with-nnunet)
```

Comment:
I guess this needs to be updated to `Transfer learning from contrast-agnostic model`

Reply:
https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/129/commits/71ab0b449fdb155eb0a930be724af152fc825b0d seems to change only the section title but the the text in the toctree
<img width="1241" alt="image" src="https://github.com/user-attachments/assets/2aeb3c9b-47a6-451c-a850-7ea4c48b6718" />

Reply:
fixed in commit https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/129/commits/78c9e03b78f466c377d2230304ca598c030dd32f